test/modules/MetricsValidator/src/tests/ValidateDocumentedMetrics.cs

```diff
@@ -42,6 +42,12 @@ protected override async Task Test(CancellationToken cancellationToken)
                 expected.Remove("edgeAgent_created_pids_total");
             }
 
+            if (OsPlatform.IsWindows())
+            {
+                // Docker doesn't return this on windows
+                expected.Remove("edgeAgent_created_pids_total");
```

Comment:
With it, will we miss a field or upload as 0? I wonder if there is any potential issue when scope script is calculated against.

Reply:
There won't be a reported metric. There are no scope scripts currently that use  this. I'm going to add this to the documentation and documentation test in my test additions